restcomm/restcomm.application/pom.xml

```diff
@@ -272,7 +272,11 @@
 			<groupId>org.hsqldb</groupId>
 			<artifactId>hsqldb</artifactId>
 		</dependency>
-
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-artifact</artifactId>
```

Comment:
@gvagenas removed, it was initially because the ExtensionConfiguration requires it